docs/IntegrationWithExistingApps.md

```diff
@@ -720,6 +720,8 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
 
 That's it, your activity is ready to run some JavaScript code.
 
+> If your app is targeting the Android `api level 23` or greater, make sure you have, for the development build, the `overlay permission` enabled. You can check it with `Settings.canDrawOverlays(this);`. This is required because, if your app produces an error in the react native component, the error view is displayed above all the other windows. Due to the new permissions system, introduced in the api level 23, the user needs to approve it.
```

Comment:
@varungupta85 the truth is that React Native supports every android version above 16, if you are targeting the >23 you need to request the permissions to the user. In the native is the same behaviour. In this PR, my point is that to show the error view, it is required the overlay permission.